src/cnd/cnd.ts

```diff
@@ -44,6 +44,50 @@ export interface SwapRequest {
   peer: Peer;
 }
 
+export interface CoreRequestBody {
+  // to_public_key: string; // This is exchanged via libp2p
+  // secret_hash: string; // Generated by cnd
+  // alpha_ledger: string; // Ledger name is in the route
+  // beta_ledger: string; // Ledger name is in the route
+  alpha_amount: string; // Asset name is in the route
+  beta_amount: string; // Asset name is in the route
+  // final_cltv_delta: number; // beta_expiry
+  // cltv_expiry: number; // beta_expiry
+  alpha_expiry: number; // Fits in JS numbers
+  beta_expiry: number; // Fits in JS numbers
+  // chain: string; // (from lnd action payload) In the URL
+  alpha_network: string; // For Ethereum we use "chain" terminology but we could revert to "network" on this new route to keep same terminology across all ledgers
+  beta_network: string; // The bitcoin network
+  // Identities are always mandatory even if cnd could discover it (for lnd).
+  // So that the interface is simpler (no optional fields) and also adds an element of security (ensure it's connected to the right lnd).
+  // Also we dropped the `redeem` and `refund` terminology as they are redundant with the `alpha/beta` qualifier.
+  // Finally, this interface is to be used by both Bob and Alice so using the `refund`/`redeem` terminology would just make it more complex
+  alpha_ledger_identity: string;
```

Comment:
:question: If we decide to keep alpha, how about this?

```suggestion
  alpha_identity: string;
```